backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserDataResolver.java

```diff
@@ -3,33 +3,53 @@
 import gov.cdc.usds.simplereport.api.InternalIdResolver;
 import gov.cdc.usds.simplereport.api.PersonNameResolver;
 import gov.cdc.usds.simplereport.db.model.ApiUser;
-import org.springframework.stereotype.Component;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+import java.util.UUID;
+import org.springframework.graphql.data.method.annotation.SchemaMapping;
+import org.springframework.stereotype.Controller;
 
 /**
  * Resolver for the graphql ApiUser type, which just (currently) contains the name of the person who
  * last modified an entity; this resolver simply aliases nameInfo to name. You may be looking for
  * {@link UserResolver}, which provides access to a much more significant graph.
  */
-@Component
+@Controller
```

Comment:
I was a bit surprised that the DataResolver is also a `@Controller` - I think `AnnotatedConfigurationController` in the [Spring for GraphQL API docs](https://docs.spring.io/spring-graphql/docs/current/api/) explains a little, but @zdeveloper if you have any more context I'd love to hear it

Reply:
Yeah, there is nothing special to the name data resolver. The notion of resolvers is not part of spring graphql, they are all controllers just like the spring restful model. With @QueryMapping and @MutationMapping following the same paradigm as @GetMapping and @PostMapping
I kept the old names because it breaks things nicely.

Reply:
Got it, thanks for the explanation!